packages/plugin-ext/src/main/browser/command-registry-main.ts

```diff
@@ -73,7 +73,18 @@ export class CommandRegistryMainImpl implements CommandRegistryMain, Disposable
 
     // tslint:disable-next-line:no-any
     $executeCommand<T>(id: string, ...args: any[]): PromiseLike<T | undefined> {
```

Comment:
again `PromiseLike` should be used only in theia.d.ts, everywhere else you can use `Promise` and `async/await`

Reply:
done